document/CoreFormat.mdk

```diff
@@ -1666,6 +1704,25 @@ and it might populate the `logicalLocations` property as follows:
 ```
 ~
 
+### `decoratedName` property { #location-decoratedName }
+
+A `location` object may contain a property named `decoratedName` whose value is a string containing
+the compiler's internal representation of the logical location associated with this `location` object.
+
+Even though `decoratedName` describes a logical location, `fullyQualifiedLogicalName` (&sect;[#location-fullyQualifiedLogicalName])
+need not be present when `decoratedName` is present.
```

Comment:
Oof. Reading this over, it makes it sound like "decoratedName" is an *alternative* to fqln. I'm going to change this to "the presence of 'decoratedName' does not imply that 'fqln' must be present".